.github/workflows/ci.yml

```diff
@@ -7,7 +7,8 @@ on:
       - '!master'
 
 env:
-  BUILDER_VERSION: v0.5.9
+  BUILDER_VERSION: more-fixes
+  BUILDER_SOURCE: channels
```

Comment:
is this something that needs to change before merge?

Reply:
yes